DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -261,6 +262,9 @@
    <version ClassVersion="11" checksum="4153489469"/>
    <version ClassVersion="10" checksum="3393361159"/>
   </class>
+  <ioread sourceClass = "pat::Jet" version="[1-17]" targetClass="pat::Jet" source="" target="jerFactorValid_">
```

Comment:
sorry, I had another chat with Slava, and I understand the read rule is not required (and thus should not be used), as the default value `jerFactorValid_(false)` is set in the constructor. Would you mind removing the read rule?

Sorry, this root IO business is not very straightforward.

Reply:
Sure, no problem. Just a moment.